ledgers/ccf/setup.py

```diff
@@ -71,7 +71,8 @@
             'ccf_ping_test=pdo.ledgers.ccf.scripts.ping_test:Main',
             'ccf_generate_ledger_authority=pdo.ledgers.ccf.scripts.generate_ledger_authority:Main',
             'ccf_fetch_ledger_authority=pdo.ledgers.ccf.scripts.fetch_ledger_authority:Main',
-            'ccf_register_enclave_policy=pdo.ledgers.ccf.scripts.register_enclave_attestation_verification_policy:Main',
+            'ccf_set_contract_enclave_check_attestation_flag=pdo.ledgers.ccf.scripts.set_contract_enclave_check_attestation_flag:Main',
+            'ccf_set_contract_enclave_expected_sgx_measurements=pdo.ledgers.ccf.scripts.set_contract_enclave_expected_sgx_measurements:Main',
         ]
```

Comment:
can we please do a better job of naming the files? come up with something that is shorter and then use both docstrings and the description in the argparse commands to provide some meaningful explanation of what the scripts do

Reply:
thanks, done